src/codegen/utilities/interpreter_helpers.py

```diff
@@ -64,14 +64,12 @@
     "GetSyncPulseTimeWhen",
     "GetTimingAttributeExTimestamp",
     "GetTimingAttributeTimestamp",
-    "GetTrigAttributeTimestamp",
```

Comment:
I implemented these functions since it seems `Start_Trig_Trig_When` uses these functions when we set or get that attribute.